### ISSUE TITLE
Construct a Polars schema or empty frame from a frame domain

### ISSUE THREAD

**Issue by Shoeboxam** [opendp/opendp#1582]
from @mccalluc:

For example, given 
```python
lf_domain = dp.lazyframe_domain([
    dp.series_domain("grade", dp.atom_domain(T=dp.i32)),
    dp.series_domain("pet_count", dp.atom_domain(T=dp.i32))]
```
it would be nice to be able to get
```python
{'grade': pl.Int32, 'pet_count': pl.Int32}
```
or 
```python
pl.LazyFrame(schema={'grade': pl.Int32, 'pet_count': pl.Int32})
```

**Comment by mccalluc** [comment]
For example, given 
```python
lf_domain = dp.lazyframe_domain([
    dp.series_domain("grade", dp.atom_domain(T=dp.i32)),
    dp.series_domain("pet_count", dp.atom_domain(T=dp.i32))]
```
it would be nice to be able to get
```python
{'grade': pl.Int32, 'pet_count': pl.Int32}
```
or 
```python
pl.LazyFrame(schema={'grade': pl.Int32, 'pet_count': pl.Int32})
```

**Comment by Shoeboxam** [comment]
Fixed in #1611
